schema_based_on_diagram.sql

```diff
@@ -0,0 +1,72 @@
+/* Database schema to keep the structure of entire database. */
+
+DROP TABLE IF EXISTS patients;
+CREATE TABLE patients (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(100),
+  date_of_birth DATE,
+  PRIMARY KEY (id)
+);
+
+DROP TABLE IF EXISTS medical_histories;
+CREATE TABLE medical_histories (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP,
+  patient_id INT,
+  PRIMARY KEY (id),
+  CONSTRAINT FK_patient_id
+  FOREIGN KEY (patient_id)
+  REFERENCES patients(id)
+);
```

Comment:
- [ ] `medical_histories` table missing `status` column from the diagram, Please make sure to add `status` column to the table.

### See the table image for reference 

![image](https://user-images.githubusercontent.com/83803414/166556720-49534748-1856-4b0a-a85e-69b51e293aad.png)